java.lang.NumberFormatException: For input string: "18,623" on TestMMCIFWriting.test1A2C

There is somme errors when we try to build the last version of the project on windows OS. For example TestMMCIFWriting.test1A2C() throws a java.lang.NumberFormatException.

On windows (java 1.8),

> INFO biojava-structure .................................. FAILURE [03:16 min] 
...
[INFO] Results:
[INFO] 
[ERROR] Errors:
[ERROR]  TestMMCIFWriting.test1A2C:70->testRoundTrip:96 » NumberFormat For input string...
[ERROR]  TestMMCIFWriting.test1SMT:53->testRoundTrip:96 » NumberFormat For input string...
[ERROR]  TestMMCIFWriting.test2N3J:64->testRoundTrip:96 » NumberFormat For input string...
[INFO] 
[ERROR] Tests run: 374, Failures: 0, Errors: 3, Skipped: 8


On cygwin os,

> java -version
openjdk version "1.8.0-292"
OpenJDK Runtime Environment (build 1.8.0-292-b10)
OpenJDK 64-Bit Server VM (build 25.71-b10, mixed mode)

Below the result of the maven package command

>  mvn package
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] biojava                                                            [pom]
[INFO] biojava-core                                                       [jar]
[INFO] biojava-alignment                                                  [jar]
[INFO] biojava-structure                                                  [jar]
[INFO] biojava-structure-gui                                              [jar]
[INFO] biojava-genome                                                     [jar]
[INFO] biojava-modfinder                                                  [jar]
[INFO] biojava-ws                                                         [jar]
[INFO] biojava-protein-disorder                                           [jar]
[INFO] biojava-aa-prop                                                    [jar]
[INFO] biojava-survival                                                   [jar]
[INFO] biojava-ontology                                                   [jar]
[INFO] biojava-protein-comparison-tool                                    [jar]
[INFO] biojava-integrationtest                                            [jar]
[INFO]
[INFO] ------------------------< org.biojava:biojava >-------------------------
[INFO] Building biojava 6.1.1-SNAPSHOT                                   [1/14]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] --- maven-enforcer-plugin:1.2:enforce (enforce-java) @ biojava ---
[INFO]
[INFO] --- maven-enforcer-plugin:1.2:enforce (enforce-maven) @ biojava ---
[INFO]
[INFO] ----------------------< org.biojava:biojava-core >----------------------
[INFO] Building biojava-core 6.1.1-SNAPSHOT                              [2/14]
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-enforcer-plugin:1.2:enforce (enforce-java) @ biojava-core ---
[INFO]
[INFO] --- maven-enforcer-plugin:1.2:enforce (enforce-maven) @ biojava-core ---
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:resources (default-resources) @ biojava-core ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] Copying 24 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:compile (default-compile) @ biojava-core ---
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] --- maven-resources-plugin:3.2.0:testResources (default-testResources) @ biojava-core ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Using 'UTF-8' encoding to copy filtered properties files.
[INFO] Copying 48 resources
[INFO]
[INFO] --- maven-compiler-plugin:3.8.1:testCompile (default-testCompile) @ biojava-core ---
[INFO] Nothing to compile - all classes are up to date
[INFO]
[INFO] --- maven-surefire-plugin:3.0.0-M5:test (default-test) @ biojava-core ---
[INFO]
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.biojava.nbio.core.alignment.matrices.AAindexFactoryTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.388 s - in org.biojava.nbio.core.alignment.matrices.AAindexFactoryTest
[INFO] Running org.biojava.nbio.core.alignment.matrices.DefaultAAIndexProviderTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.385 s - in org.biojava.nbio.core.alignment.matrices.DefaultAAIndexProviderTest
[INFO] Running org.biojava.nbio.core.alignment.matrices.SubstitutionMatrixHelperTest
[INFO] Tests run: 23, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.409 s - in org.biojava.nbio.core.alignment.matrices.SubstitutionMatrixHelperTest
[INFO] Running org.biojava.nbio.core.alignment.matrices.SubstitutionMatrixTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.338 s - in org.biojava.nbio.core.alignment.matrices.SubstitutionMatrixTest
[INFO] Running org.biojava.nbio.core.alignment.SimpleAlignedSequenceTest
[WARNING] Tests run: 35, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.329 s - in org.biojava.nbio.core.alignment.SimpleAlignedSequenceTest
[INFO] Running org.biojava.nbio.core.alignment.SimpleProfileTest
[WARNING] Tests run: 54, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.362 s - in org.biojava.nbio.core.alignment.SimpleProfileTest
[INFO] Running org.biojava.nbio.core.alignment.SimpleSequencePairTest
[INFO] Tests run: 35, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.337 s - in org.biojava.nbio.core.alignment.SimpleSequencePairTest
[INFO] Running org.biojava.nbio.core.fasta.TestFASTAReader
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.338 s - in org.biojava.nbio.core.fasta.TestFASTAReader
[INFO] Running org.biojava.nbio.core.search.io.blast.BlastTabularParserTest
createObjects
getFileExtensions
setFile
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.77 s - in org.biojava.nbio.core.search.io.blast.BlastTabularParserTest
[INFO] Running org.biojava.nbio.core.search.io.blast.BlastXMLParserTest
createObjects
getFileExtensions
setFile
[WARNING] Tests run: 6, Failures: 0, Errors: 0, Skipped: 2, Time elapsed: 1.837 s - in org.biojava.nbio.core.search.io.blast.BlastXMLParserTest
[INFO] Running org.biojava.nbio.core.search.io.HspTest
getAlignment
hashCode
equals
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.325 s - in org.biojava.nbio.core.search.io.HspTest
[INFO] Running org.biojava.nbio.core.search.io.SearchIOTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 3.368 s - in org.biojava.nbio.core.search.io.SearchIOTest
[INFO] Running org.biojava.nbio.core.sequence.ChromosomeSequenceTest
[INFO] Running org.biojava.nbio.core.sequence.ChromosomeSequenceTest$AddingAndRemovingGeneSequences
[INFO] Tests run: 20, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.008 s - in org.biojava.nbio.core.sequence.ChromosomeSequenceTest$AddingAndRemovingGeneSequences
[INFO] Running org.biojava.nbio.core.sequence.ChromosomeSequenceTest$AfterConstructionWithEmptyString
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.003 s - in org.biojava.nbio.core.sequence.ChromosomeSequenceTest$AfterConstructionWithEmptyString
[INFO] Running org.biojava.nbio.core.sequence.ChromosomeSequenceTest$AfterValidConstruction
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.004 s - in org.biojava.nbio.core.sequence.ChromosomeSequenceTest$AfterValidConstruction
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.405 s - in org.biojava.nbio.core.sequence.ChromosomeSequenceTest
[INFO] Running org.biojava.nbio.core.sequence.compound.AmbiguityDNACompoundTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.302 s - in org.biojava.nbio.core.sequence.compound.AmbiguityDNACompoundTest
[INFO] Running org.biojava.nbio.core.sequence.DNATest
[INFO] Tests run: 20, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.363 s - in org.biojava.nbio.core.sequence.DNATest
[INFO] Running org.biojava.nbio.core.sequence.EditSequenceTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.321 s - in org.biojava.nbio.core.sequence.EditSequenceTest
[INFO] Running org.biojava.nbio.core.sequence.ExonComparatorTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.337 s - in org.biojava.nbio.core.sequence.ExonComparatorTest
[INFO] Running org.biojava.nbio.core.sequence.ExonSequenceTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.329 s - in org.biojava.nbio.core.sequence.ExonSequenceTest
[INFO] Running org.biojava.nbio.core.sequence.GeneSequenceTest
[INFO] Running org.biojava.nbio.core.sequence.GeneSequenceTest$AfterValidConstruction
[WARNING] Tests run: 11, Failures: 0, Errors: 0, Skipped: 2, Time elapsed: 0.006 s - in org.biojava.nbio.core.sequence.GeneSequenceTest$AfterValidConstruction
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.349 s - in org.biojava.nbio.core.sequence.GeneSequenceTest
[INFO] Running org.biojava.nbio.core.sequence.io.ABITracerTest
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.693 s - in org.biojava.nbio.core.sequence.io.ABITracerTest
[INFO] Running org.biojava.nbio.core.sequence.io.CasePreservingProteinSequenceCreatorTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.338 s - in org.biojava.nbio.core.sequence.io.CasePreservingProteinSequenceCreatorTest
[INFO] Running org.biojava.nbio.core.sequence.io.embl.EmblReaderTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.023 s - in org.biojava.nbio.core.sequence.io.embl.EmblReaderTest
[INFO] Running org.biojava.nbio.core.sequence.io.FastaGeneWriterTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.336 s - in org.biojava.nbio.core.sequence.io.FastaGeneWriterTest
[INFO] Running org.biojava.nbio.core.sequence.io.FastaReaderTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.342 s - in org.biojava.nbio.core.sequence.io.FastaReaderTest
[INFO] Running org.biojava.nbio.core.sequence.io.FastaWriterTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.316 s - in org.biojava.nbio.core.sequence.io.FastaWriterTest
[INFO] Running org.biojava.nbio.core.sequence.io.GenbankCookbookTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.904 s <<< FAILURE! - in org.biojava.nbio.core.sequence.io.GenbankCookbookTest
[ERROR] org.biojava.nbio.core.sequence.io.GenbankCookbookTest.testProcess  Time elapsed: 0.595 s  <<< ERROR!
java.net.UnknownHostException: eutils.ncbi.nlm.nih.gov
        at org.biojava.nbio.core.sequence.io.GenbankCookbookTest.testProcess(GenbankCookbookTest.java:74)

[INFO] Running org.biojava.nbio.core.sequence.io.GenbankReaderTest
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.359 s - in org.biojava.nbio.core.sequence.io.GenbankReaderTest
[INFO] Running org.biojava.nbio.core.sequence.io.GenbankWriterTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.358 s - in org.biojava.nbio.core.sequence.io.GenbankWriterTest
[INFO] Running org.biojava.nbio.core.sequence.io.GenericFastaHeaderParserTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.307 s - in org.biojava.nbio.core.sequence.io.GenericFastaHeaderParserTest
[INFO] Running org.biojava.nbio.core.sequence.JoiningSequenceReaderTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.311 s - in org.biojava.nbio.core.sequence.JoiningSequenceReaderTest
[INFO] Running org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReaderTest
00:27:19 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id CP001663.1. Error: eutils.ncbi.nlm.nih.gov
00:27:19 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id CP001663.1. Error: eutils.ncbi.nlm.nih.gov
00:27:19 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id NC_000932.1. Error: eutils.ncbi.nlm.nih.gov
00:27:19 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id NC_000932.1. Error: eutils.ncbi.nlm.nih.gov
00:27:19 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id NC_000932.1. Error: eutils.ncbi.nlm.nih.gov
00:27:19 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id NC_000932.1. Error: eutils.ncbi.nlm.nih.gov
00:27:19 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id NC_000932.1. Error: eutils.ncbi.nlm.nih.gov
00:27:19 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id NC_000932.1. Error: eutils.ncbi.nlm.nih.gov
00:27:20 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id NC_016912.1. Error: eutils.ncbi.nlm.nih.gov
00:27:20 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id NC_016912.1. Error: eutils.ncbi.nlm.nih.gov
00:27:22 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id NC_009648.1. Error: eutils.ncbi.nlm.nih.gov
00:27:22 [main] ERROR org.biojava.nbio.core.sequence.ProteinSequence - Caught IOException when getting DNA sequence for id NC_009648.1. Error: eutils.ncbi.nlm.nih.gov
[ERROR] Tests run: 16, Failures: 1, Errors: 6, Skipped: 0, Time elapsed: 5.491 s <<< FAILURE! - in org.biojava.nbio.core.sequence.loader.GenbankProxySequenceReaderTest
[ERROR] GenbankProxySequenceReaderTest.testProteinSequenceFactoring  Time elapsed: 0.558 s  <<< ERROR!
java.lang.NullPointerException

[ERROR] GenbankProxySequenceReaderTest.testFeatures  Time elapsed: 0.009 s  <<< ERROR!
java.lang.NullPointerException

[ERROR] GenbankProxySequenceReaderTest.testProteinSequenceFactoring  Time elapsed: 0.012 s  <<< FAILURE!
java.lang.AssertionError

[ERROR] GenbankProxySequenceReaderTest.testProteinSequenceFactoring  Time elapsed: 0.028 s  <<< ERROR!
java.lang.NullPointerException

[ERROR] GenbankProxySequenceReaderTest.testFeatures  Time elapsed: 0.014 s  <<< ERROR!
java.lang.NullPointerException

[ERROR] GenbankProxySequenceReaderTest.testProteinSequenceFactoring  Time elapsed: 0.029 s  <<< ERROR!
java.lang.NullPointerException

[ERROR] GenbankProxySequenceReaderTest.testFeatures  Time elapsed: 0.019 s  <<< ERROR!
java.lang.NullPointerException

[INFO] Running org.biojava.nbio.core.sequence.loader.SimpleGenbankProxySequenceReaderTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.668 s - in org.biojava.nbio.core.sequence.loader.SimpleGenbankProxySequenceReaderTest
[INFO] Running org.biojava.nbio.core.sequence.location.InsdcParserTest
[INFO] Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.395 s - in org.biojava.nbio.core.sequence.location.InsdcParserTest
[INFO] Running org.biojava.nbio.core.sequence.location.LocationParserTest
[WARNING] Tests run: 2, Failures: 0, Errors: 0, Skipped: 2, Time elapsed: 0.008 s - in org.biojava.nbio.core.sequence.location.LocationParserTest
[INFO] Running org.biojava.nbio.core.sequence.location.LocationTest
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.313 s - in org.biojava.nbio.core.sequence.location.LocationTest
[INFO] Running org.biojava.nbio.core.sequence.location.TargetedLocationParserTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.021 s - in org.biojava.nbio.core.sequence.location.TargetedLocationParserTest
[INFO] Running org.biojava.nbio.core.sequence.MultipleSequenceAlignmentTest
[INFO] Tests run: 11, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.383 s - in org.biojava.nbio.core.sequence.MultipleSequenceAlignmentTest
[INFO] Running org.biojava.nbio.core.sequence.ProteinSequenceTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.325 s - in org.biojava.nbio.core.sequence.ProteinSequenceTest
[INFO] Running org.biojava.nbio.core.sequence.RNASequenceTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.352 s - in org.biojava.nbio.core.sequence.RNASequenceTest
[INFO] Running org.biojava.nbio.core.sequence.SequenceViewTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.311 s - in org.biojava.nbio.core.sequence.SequenceViewTest
[INFO] Running org.biojava.nbio.core.sequence.StrandTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.032 s - in org.biojava.nbio.core.sequence.StrandTest
[INFO] Running org.biojava.nbio.core.sequence.TaxonomyIDTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.031 s - in org.biojava.nbio.core.sequence.TaxonomyIDTest
[INFO] Running org.biojava.nbio.core.sequence.TestSequenceEquals
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.31 s - in org.biojava.nbio.core.sequence.TestSequenceEquals
[INFO] Running org.biojava.nbio.core.sequence.TranscriptSequenceTest
[INFO] Running org.biojava.nbio.core.sequence.TranscriptSequenceTest$AfterValidConstruction
[WARNING] Tests run: 11, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.008 s - in org.biojava.nbio.core.sequence.TranscriptSequenceTest$AfterValidConstruction
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.353 s - in org.biojava.nbio.core.sequence.TranscriptSequenceTest
[INFO] Running org.biojava.nbio.core.sequence.TranslationTest
[INFO] Tests run: 12, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.464 s - in org.biojava.nbio.core.sequence.TranslationTest
[INFO] Running org.biojava.nbio.core.sequence.views.WindowViewTests
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.309 s - in org.biojava.nbio.core.sequence.views.WindowViewTests
[INFO] Running org.biojava.nbio.core.TestAmbiguityCompoundSet
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.346 s - in org.biojava.nbio.core.TestAmbiguityCompoundSet
[INFO] Running org.biojava.nbio.core.util.CRC64ChecksumTest
[INFO] Tests run: 17, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.062 s - in org.biojava.nbio.core.util.CRC64ChecksumTest
[INFO] Running org.biojava.nbio.core.util.EqualsTest
[INFO] Running org.biojava.nbio.core.util.EqualsTest$ClassEquals
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.016 s - in org.biojava.nbio.core.util.EqualsTest$ClassEquals
[INFO] Running org.biojava.nbio.core.util.EqualsTest$ObjectEquals
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.004 s - in org.biojava.nbio.core.util.EqualsTest$ObjectEquals
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.056 s - in org.biojava.nbio.core.util.EqualsTest
[INFO] Running org.biojava.nbio.core.util.FileDownloadUtilsTest
[INFO] Running org.biojava.nbio.core.util.FileDownloadUtilsTest$DeleteDirectory
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.707 s - in org.biojava.nbio.core.util.FileDownloadUtilsTest$DeleteDirectory
[INFO] Running org.biojava.nbio.core.util.FileDownloadUtilsTest$URLMethods
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.353 s - in org.biojava.nbio.core.util.FileDownloadUtilsTest$URLMethods
[INFO] Running org.biojava.nbio.core.util.FileDownloadUtilsTest$ExpandUserHome
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.012 s - in org.biojava.nbio.core.util.FileDownloadUtilsTest$ExpandUserHome
[INFO] Running org.biojava.nbio.core.util.FileDownloadUtilsTest$ToUnixPath
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.006 s - in org.biojava.nbio.core.util.FileDownloadUtilsTest$ToUnixPath
[INFO] Running org.biojava.nbio.core.util.FileDownloadUtilsTest$GetFilePrefix
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.004 s - in org.biojava.nbio.core.util.FileDownloadUtilsTest$GetFilePrefix
[INFO] Running org.biojava.nbio.core.util.FileDownloadUtilsTest$FileExtension
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.005 s - in org.biojava.nbio.core.util.FileDownloadUtilsTest$FileExtension
[INFO] Running org.biojava.nbio.core.util.FileDownloadUtilsTest$FileCopy
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.006 s - in org.biojava.nbio.core.util.FileDownloadUtilsTest$FileCopy
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.118 s - in org.biojava.nbio.core.util.FileDownloadUtilsTest
[INFO] Running org.biojava.nbio.core.util.FlatFileCacheTest
[INFO] Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.768 s - in org.biojava.nbio.core.util.FlatFileCacheTest
[INFO] Running org.biojava.nbio.core.util.HashcoderTest
[INFO] Tests run: 21, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.07 s - in org.biojava.nbio.core.util.HashcoderTest
[INFO] Running org.biojava.nbio.core.util.PrettyXMLWriterTest
<to>
  <a>22</a>
  <b>43.22</b>
  <text>some text</text>
  <dt tz="UTC">2021-08-15</dt>
</to>

[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.326 s - in org.biojava.nbio.core.util.PrettyXMLWriterTest
[INFO] Running org.biojava.nbio.core.util.SequenceToolsTest
[INFO] Running org.biojava.nbio.core.util.SequenceToolsTest$SequenceFromString
[INFO] Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.361 s - in org.biojava.nbio.core.util.SequenceToolsTest$SequenceFromString
[INFO] Running org.biojava.nbio.core.util.SequenceToolsTest$PercentNucleotideContent
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.038 s - in org.biojava.nbio.core.util.SequenceToolsTest$PercentNucleotideContent
[INFO] Running org.biojava.nbio.core.util.SequenceToolsTest$PermuteCyclic
[WARNING] Tests run: 8, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.036 s - in org.biojava.nbio.core.util.SequenceToolsTest$PermuteCyclic
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.449 s - in org.biojava.nbio.core.util.SequenceToolsTest
[INFO] Running org.biojava.nbio.core.util.SingleLinkageClustererTest
[INFO] Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.334 s - in org.biojava.nbio.core.util.SingleLinkageClustererTest
[INFO] Running org.biojava.nbio.core.util.SoftHashMapTest
[WARNING] Tests run: 3, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.326 s - in org.biojava.nbio.core.util.SoftHashMapTest
[INFO] Running org.biojava.nbio.core.util.StringManipulationHelperTest
[INFO] Running org.biojava.nbio.core.util.StringManipulationHelperTest$EqualsToXml
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.322 s - in org.biojava.nbio.core.util.StringManipulationHelperTest$EqualsToXml
[INFO] Running org.biojava.nbio.core.util.StringManipulationHelperTest$JoinString
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.007 s - in org.biojava.nbio.core.util.StringManipulationHelperTest$JoinString
[INFO] Running org.biojava.nbio.core.util.StringManipulationHelperTest$equalsToIgnoreEndline
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.009 s - in org.biojava.nbio.core.util.StringManipulationHelperTest$equalsToIgnoreEndline
[INFO] Running org.biojava.nbio.core.util.StringManipulationHelperTest$InputStreamToString
[INFO] Tests run: 6, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.04 s - in org.biojava.nbio.core.util.StringManipulationHelperTest$InputStreamToString
[INFO] Running org.biojava.nbio.core.util.StringManipulationHelperTest$PaddingTest
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.008 s - in org.biojava.nbio.core.util.StringManipulationHelperTest$PaddingTest
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.402 s - in org.biojava.nbio.core.util.StringManipulationHelperTest
[INFO] Running org.biojava.nbio.core.util.TestUncompressInputStream
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.309 s - in org.biojava.nbio.core.util.TestUncompressInputStream
[INFO] Running org.biojava.nbio.core.util.XMLHelperTest
[INFO] Running org.biojava.nbio.core.util.XMLHelperTest$SelectElements
[INFO] Tests run: 10, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.024 s - in org.biojava.nbio.core.util.XMLHelperTest$SelectElements
[INFO] Running org.biojava.nbio.core.util.XMLHelperTest$SelectSingleElement
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.006 s - in org.biojava.nbio.core.util.XMLHelperTest$SelectSingleElement
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.794 s - in org.biojava.nbio.core.util.XMLHelperTest
[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR] GenbankProxySequenceReaderTest.testFeatures
[ERROR]   Run 1: null
[INFO]   Run 2: PASS
[ERROR]   Run 3: null
[INFO]   Run 4: PASS
[INFO]   Run 5: PASS
[ERROR]   Run 6: null
[INFO]   Run 7: PASS
[INFO]   Run 8: PASS
[INFO]
[ERROR] GenbankProxySequenceReaderTest.testProteinSequenceFactoring
[ERROR]   Run 1: null
[ERROR]   Run 2: null
[ERROR]   Run 3: null
[INFO]   Run 4: PASS
[INFO]   Run 5: PASS
[ERROR]   Run 6: null
[INFO]   Run 7: PASS
[INFO]   Run 8: PASS
[INFO]
[ERROR]   GenbankCookbookTest.testProcess:74 » UnknownHost eutils.ncbi.nlm.nih.gov
[INFO]
[ERROR] Tests run: 487, Failures: 0, Errors: 3, Skipped: 11
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for biojava 6.1.1-SNAPSHOT:
[INFO]
[INFO] biojava ............................................ SUCCESS [  0.361 s]
[INFO] biojava-core ....................................... FAILURE [01:06 min]
[INFO] biojava-alignment .................................. SKIPPED
[INFO] biojava-structure .................................. SKIPPED
[INFO] biojava-structure-gui .............................. SKIPPED
[INFO] biojava-genome ..................................... SKIPPED
[INFO] biojava-modfinder .................................. SKIPPED
[INFO] biojava-ws ......................................... SKIPPED
[INFO] biojava-protein-disorder ........................... SKIPPED
[INFO] biojava-aa-prop .................................... SKIPPED
[INFO] biojava-survival ................................... SKIPPED
[INFO] biojava-ontology ................................... SKIPPED
[INFO] biojava-protein-comparison-tool .................... SKIPPED
[INFO] biojava-integrationtest ............................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:07 min
[INFO] Finished at: 2023-01-08T00:27:50+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test (default-test) on project biojava-core: There are test failures.
[ERROR]
[ERROR] Please refer to D:\dev\os-projects\biojava\biojava-core\target\surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :biojava-core

> by changing the locale to something that doesn't have the UK/US number format

I think you are right but I don't have the full stacktrace.

> [ERROR] org.biojava.nbio.structure.io.TestMMCIFWriting.test1A2C  Time elapsed: 1.504 s  <<< ERROR!
java.lang.NumberFormatException: For input string: "18,623"
	at org.biojava.nbio.structure.io.TestMMCIFWriting.testRoundTrip(TestMMCIFWriting.java:96)
	at org.biojava.nbio.structure.io.TestMMCIFWriting.test1A2C(TestMMCIFWriting.java:70)
	
> [ERROR] org.biojava.nbio.structure.io.TestMMCIFWriting.test1SMT  Time elapsed: 0.05 s  <<< ERROR!
java.lang.NumberFormatException: For input string: "-6,786"
	at org.biojava.nbio.structure.io.TestMMCIFWriting.testRoundTrip(TestMMCIFWriting.java:96)
	at org.biojava.nbio.structure.io.TestMMCIFWriting.test1SMT(TestMMCIFWriting.java:53)
	
> [ERROR] org.biojava.nbio.structure.io.TestMMCIFWriting.test2N3J  Time elapsed: 0.46 s  <<< ERROR!
java.lang.NumberFormatException: For input string: "12,789"
	at org.biojava.nbio.structure.io.TestMMCIFWriting.testRoundTrip(TestMMCIFWriting.java:96)
	at org.biojava.nbio.structure.io.TestMMCIFWriting.test2N3J(TestMMCIFWriting.java:64)

--- a/pom.xml
+++ b/pom.xml
@@ -45,7 +45,7 @@
 		<log4j.version>2.17.2</log4j.version>
 		<junit-jupiter.version>5.7.2</junit-jupiter.version>
 		<ciftools.artifact>ciftools-java-jdk8</ciftools.artifact>
-		<ciftools.version>4.0.3</ciftools.version>
+		<ciftools.version>4.0.5</ciftools.version>
 	</properties>
 	<scm>
 		<connection>scm:git:git://github.com/biojava/biojava.git</connection>

